fetch.bs

```diff
@@ -2304,18 +2304,28 @@ known as an <dfn export id=concept-aborted-network-error>aborted network error</
 
 <hr>
 
-<p>A <dfn export id=concept-filtered-response>filtered response</dfn> is a limited view on a
-<a for=/>response</a> that is not a
-<a>network error</a>. This
-<a for=/>response</a> is referred to as the
-<a>filtered response</a>'s associated
-<dfn export id=concept-internal-response for="filtered response">internal response</dfn>.
-
-<p class="note no-backref">The <a for=/>fetch</a> algorithm returns such a view to ensure APIs do
-not accidentally leak information. If the information needs to be exposed for legacy reasons, e.g.,
-to feed image data to a decoder, the associated <a for="filtered response">internal response</a> can
-be used, which is only "accessible" to internal specification algorithms and is never a
-<a>filtered response</a> itself.
+<p>A <dfn export id=concept-filtered-response>filtered response</dfn> is a <a for=/>response</a>
+that offers a limited view on an associated <a for=/>response</a>. This associated
+<a for=/>response</a> can be accessed through <a>filtered response</a>'s associated
+<dfn export id=concept-internal-response for="filtered response">internal response</dfn> (a
+<a for=/>response</a> that is neither a <a for=/>network error</a> nor a
+<a for=/>filtered response</a>).
+
+<p>Unless stated otherwise a <a for=/>filtered response</a>'s associated concepts (such as its
+<a for=response>body</a>) refer to the associated concepts of its
+<a for="filtered response">internal response</a>. (The exceptions to this are listed below as part
+of defining the concrete types of <a for=/>filtered responses</a>.)
+
+<div class=note>
+ <p>The <a for=/>fetch</a> algorithm returns such a view to ensure APIs do not accidentally leak
+ information. If the information needs to be exposed for legacy reasons, e.g., to feed image data to
+ a decoder, the associated <a for="filtered response">internal response</a> can be used, which is
+ only "accessible" to internal specification algorithms.
```

Comment:
What internal specification algorithms is it accessible to?

Reply:
Any that need access? E.g., scripts, images, style sheets.

Reply:
I stil find this sentence confusing them... what is an example of a non-internal specification algorithm, which doesn't have access?

Reply:
Oh, I could remove the word "internal".